product_docs/docs/pem/8/tuning_performance/tuning_wizard.mdx

```diff
@@ -8,84 +8,88 @@ redirects:
 
-Click `Next` to continue to the `Tuning Changes Summary` dialog.
+Select **Next** to continue.
 
 ![The Tuning Changes Summary dialog](../images/tuning_wiz_changes_sum.png)
 
-The tree control on the `Tuning Changes Summary` dialog displays the parameter setting modifications recommended for each server analyzed by the Tuning Wizard. Use the checkboxes next to a server or parameter name to select the recommendations that tuning wizard will either include in a preview report or apply:
+The tree menu on the **Tuning Changes Summary** window displays the parameter setting modifications recommended for each server analyzed by the Tuning Wizard. Select the recommendations that you want Tuning Wizard to apply or to include in a preview report:
 
--   A checked box to the left of a parameter name specifies that the Tuning Wizard will include the parameter setting.
--   A checked box to the left of a server name specifies that the Tuning Wizard will include all parameter setting recommendations for the specified server.
+-   Select a parameter name and Tuning Wizard includes the parameter setting.
+-   Select the server name and the Tuning Wizard includes all parameter setting recommendations for the specified server.
 
-Specify which Tuning Wizard recommendations you wish to include in a report or apply, and click `Next` to continue.
+Select **Next** to continue.
 
-Use the `Schedule or Run?` dialog to either specify a time that PEM will apply the changes, or generate a report that details the recommended changes.
+In the **Schedule or Run?** window, either choose a time when PEM applies the recommended changes or choose to generate a report that details the recommended changes.
 
-The selected actions will apply to all of the changes noted on the Tuning Changes Summary. If you opt to generate a report, PEM will create a report that contains a list of the current values and recommended modifications to the configuration parameters selected on the Tuning Changes Summary dialog. Note that to implement changes, you will need to invoke the Tuning Wizard a second time, specifying the parameters you wish to modify on the `Tuning Changes Summary` dialog.
+PEM makes the recommended changes that you selected in the **Tuning Changes Summary** window. If you choose to generate a report, then PEM creates a report. It contains a list of the current values and recommended modifications to the configuration parameters as selected in the **Tuning Changes Summary** window. To implement changes, you must open the Tuning Wizard a second time, selecting the parameters you want to modify in the **Tuning Changes Summary** window.
 
-Select `Schedule changes` to view and specify your scheduling options.
+Select **Schedule changes** to view and specify your scheduling options.
 
 ![The Schedule or Run? dialog](../images/tuning_wiz_apply_changes.png)
 
-You can:
+The **Configuration now?** slider can be set to:
```

Comment:
This is a nice set of edits. But this edit made the phrase passive voice. So:
You can set the ... slider to: